### PR TITLE
Merge editorialized release notes for 17.7

### PR DESCRIPTION
**Pull request by mokagio**
Merges the release notes so they can be translated in GlotPress.

### CODE REVIEW

File: WordPress/Jetpack/Resources/release_notes.txt
Comment (mokagio) [inline WordPress/Jetpack/Resources/release_notes.txt:1]: @leandroalonso I wasn't 100% sure from the wording in the editorial P2 post whether this is the desired copy for the release notes.

Can you please tell me if I got it right?

File: WordPress/Resources/AppStoreStrings.po
Comment (mokagio) [inline WordPress/Resources/AppStoreStrings.po:45]: Aside: it's funny that the word by word diff shows a certain pattern "you ... the ... to ... the ..."

The English language really does have certain common structures and frequent words 🤓

**Review by oguzkocer — APPROVED** [review]
_This is not the "Extract Jetpack app identifier to constants in Fastfile" PR 👀 😕_ 

Everything seems in place, but I'll leave it to @leandroalonso to give the 👍 for content.

**Comment by mokagio** [comment]
> This is not the "Extract Jetpack app identifier to constants in Fastfile" PR 👀 😕

Maybe it was a GitHub cache issue? I renamed it to "Merge editorialized release notes for 17.7" right after asking for review 🤔 

image

(The reason it ended up with that title is that, for a change, I used the GitHub UI to make the PR instead of `hub` and was too hasty in opening it, forgetting to fix the title. Because this PR comes on top of `release/17.7` and GitHub opens PRs against the main branch by default, GitHub used the title of the first commit diverging from `develop` as the title and I didn't notice.)

**Comment by mokagio** [comment]
Thanks @oguzkocer 🙇‍♂️ 

I just want to wait for @leandroalonso +1 on the notes before merging to avoid having to ask the translators to update their copy in case I misinterpreted the editorialized post.

File: WordPress/Jetpack/Resources/release_notes.txt
Comment (leandroalonso) [reply WordPress/Jetpack/Resources/release_notes.txt:1]: Yes, it is.

Blogging Reminders is also present but we had an issue with it so I removed it from the copy, just in case. The issue is fixed now, but it will be out of the release notes (it's not merged yet anyway).

**Pull request merged**
